src/audio_worklet_node.rs

```diff
@@ -51,6 +51,8 @@ struct ProcessorArguments {
 struct ProcessCallChannel {
     send: Sender<WorkletCommand>,
     recv: Receiver<WorkletCommand>,
+    // mark that the worklet has been exited to prevent any further `process` call
```

Comment:
Cool to have a clean exit now. I will indeed try to butcher this a bit, I think this will be better modeled with a new type of WorkletCommand instead, to solve any `recv()` call hanging

Reply:
Looking better into this, I think your solution is just good as it is.
We can't use the WorkletCommand because it is strictly from rust audio thread to js worker, and here we need to move information vice versa. I don't see a way to further simplify this.